testing/trino-product-tests-groups/src/main/java/io/trino/tests/product/TestGroups.java

```diff
@@ -46,7 +46,7 @@ public final class TestGroups
     public static final String OAUTH2_REFRESH = "oauth2_refresh";
     public static final String MYSQL = "mysql";
     public static final String TRINO_JDBC = "trino_jdbc";
-    public static final String QUERY_ENGINE = "qe";
+    public static final String QE = "qe"; // query engine
```

Comment:
can you rename to QUERY_ENGINE as a followup?

Reply:
i didn't want to do replacements in convention based tests, because it's more erronous (no static checks).

the convention based need some cleanup and/or pruning (eg https://github.com/trinodb/trino/pull/20352)
let's leave it for a follow-up